roles/lib_openshift/src/test/unit/conftest.py

```diff
@@ -0,0 +1,55 @@
+import inspect
+
+import pytest
+
+from lib_openshift.library import *  # noqa: F401,F403
```

Comment:
@ashcrow indeed, I'm trying to strike a balance between showing accurate coverage for the generated code without duplicating tests. I'll try to massage it up a bit more.